src/main/kotlin/com/ex_dock/ex_dock/database/account/AccountJdbcVerticle.kt

```diff
@@ -1,47 +1,55 @@
 package com.ex_dock.ex_dock.database.account
 
 import com.ex_dock.ex_dock.database.connection.Connection
+import io.vertx.core.AbstractVerticle
 import io.vertx.core.eventbus.EventBus
-import io.vertx.core.json.JsonObject
-import io.vertx.ext.web.RoutingContext
-import io.vertx.kotlin.core.json.Json
 import io.vertx.kotlin.core.json.json
 import io.vertx.kotlin.core.json.obj
-import io.vertx.kotlin.coroutines.CoroutineVerticle
-import io.vertx.kotlin.coroutines.coAwait
 import io.vertx.sqlclient.Pool
+import io.vertx.sqlclient.Row
+import io.vertx.sqlclient.RowSet
 
-class AccountJdbcVerticle: CoroutineVerticle() {
+class AccountJdbcVerticle: AbstractVerticle() {
 
   private lateinit var client: Pool
 
   private lateinit var eventBus: EventBus
 
-  override suspend fun start() {
+  override fun start() {
     client = Connection().getConnection(vertx)
     eventBus = vertx.eventBus()
 
     getAccountData()
   }
 
-  private suspend fun getAll(ctx: RoutingContext) {
-    val rows = client.preparedQuery("SELECT * FROM importance").execute().coAwait()
-    if (rows.size() > 0) {
-      ctx.response().end(json {
-        obj("importance_levels" to rows.iterator().next().getInteger("importance_levels")).encode()
-      })
-    } else {
-      ctx.response().setStatusCode(404).end()
-    }
-  }
-
   private fun getAccountData() {
-    val consumer = eventBus.localConsumer<String>("process.account.getData")
+    val consumer = eventBus.localConsumer<Any>("process.account.getData")
```

Comment:
The variable consumer isn't very clear about what it is doing. a name like "consumerEventBus" would be clearer about it's purpose.

Reply:
I changed the type to JsonObject. The Any was just for testing purposes, but I forget to change them.